### ISSUE TITLE
Error when running from a yarn script

### ISSUE THREAD

**Issue by maxfi** [Plazide/fauna-gql-upload#6]
With the following yarn script:

```json
{
  "scripts": {
    "fauna-gql-upload": "fauna-gql"
  },
}
```

when running `yarn run fauna-gql-upload` the following error is produced:

```
/usr/bin/env: ‘node\r’: No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

This is an [issue with yarn](https://github.com/yarnpkg/yarn/issues/8106). 

Converting https://github.com/Plazide/fauna-gql-upload/blob/master/index.js#L1 to use Unix line endings (`LF` rather than `CRLF`) fixes the issue.

I can create a pull request if you agree. Thanks.

**Comment by Plazide** [comment]
Sure, create a pull request. 

Probably best to just use `LF` for all files.

Never encountered this before, so good catch!

**Comment by maxfi** [comment]
So this is kinda weird...

When I `yarn install` the package I get the following:

```
node_modules/fauna-gql-upload$ file index.js 
index.js: a /usr/bin/env node script, ASCII text executable, with CRLF line terminators
```

But when I clone this repo I get:
```
fauna-gql-upload$ file index.js 
index.js: a /usr/bin/env node script, ASCII text executable
```

Any ideas? Is there something happening when you publish the package?

**Comment by Plazide** [comment]
Yeah, that is strange.

I did the same check as you did above, but got the opposite results.

In the project folder I get:
![bild](https://user-images.githubusercontent.com/20641118/89710862-cc9e6880-d986-11ea-8457-8f8582b259e6.png)

But when the package is installed I get:
![bild](https://user-images.githubusercontent.com/20641118/89710903-2ef76900-d987-11ea-8df1-a5cb5cdcfb34.png)

Don't really know what to make of this. I don't do anything special when publishing to npm, just `npm publish`. I thought maybe it's the tarball created by npm that changes the line endings, but then the line endings should be the same after installing.

Could possibly be another issue with yarn. Which version are you using? I'm using `1.19.1`.

**Comment by maxfi** [comment]
Sorry about the late reply. Yeah, that's really strange...

Do you get the error:

```
/usr/bin/env: ‘node\r’: No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

when running from a yarn script?

I'm using yarn `1.22.4`. Vercel also uses the same version and I'm getting the same error on deployments there.

**Comment by Plazide** [comment]
I don't get that error, but I'll try and update yarn to your version and see if it happens.

**Comment by Plazide** [comment]
So, I updated yarn. Still not getting an error. Though I noticed that when installing with version `1.22.4`, the installed package file returns 

```sh
node_modules/fauna-gql-upload/index.js: a /usr/bin/env node script, ASCII text executable, with CRLF line terminators
```
The same as yours. So, that seems to be a difference across yarn versions.

I can't seem to reproduce the error, either on windows or ubuntu. I'm using WSL, so I even tried `yarn install` on windows and then running `yarn run fauna-gql-upload` on the Ubuntu distro. Still no error.

Could you share a repo to reproduce the error? Even though changing the line endings solves it, I'd still like to know what the cause is.

**Comment by maxfi** [comment]
Very strange...

I've found this issue: https://github.com/yarnpkg/yarn/issues/5480
and from there a link to here: https://github.com/sainthkh/reasonql/issues/14#issuecomment-529600513

I'll investigate I bit further tomorrow and try to create a minimal reproduction. Will keep you posted. Thanks.

**Comment by maxfi** [comment]
@Plazide I can't figure this out... What do you think about adding the `.gitattributes` file that sets the line endings and then publishing a patch version to test if it worked?

**Comment by Plazide** [comment]
Sure, let's try that. Create the PR, I'll merge, and then publish a dev version on npm.

**Comment by Plazide** [comment]
@maxfi I added the `.gitattributes` file and published a new version to npm with a dev tag. Try `yarn add fauna-gql-upload@1.6.1` and tell me if it works for you.

**Comment by maxfi** [comment]
@Plazide any ideas how I can troubleshoot this? Thanks.

**Comment by Plazide** [comment]
@maxfi I really have no idea.

Do you have the same problem with every version of the package? Or is it just the later versions?

**Comment by Plazide** [comment]
@maxfi I published another version yesterday, `fauna-gql-upload@1.6.3`. Try installing that, but maybe delete your `node_modules` before you do that. 

I changed some local git settings, specifically `core.autocrlf`. It was set to `true` which means git converts all line-endings to CRLF. I've changed this setting to `input`, which is the recommended setting on Unix based systems.

Hopefully, it will help.
